java/alizer-api/pom.xml

```diff
@@ -36,4 +36,38 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <version>3.2.0</version>
```

Comment:
The version is fixed by parent from JBoss organization. Also pluginManagement is for heritance so it is more readable to specify the plugin configuration in <build><plugins> section